commit 10e5ea6d3390b4dcfd7eea8a54b3396ba222cb5a (refs/changes/57/357/7)
Author: Brian Bennett <brian.bennett@joyent.com>
Date:   2016-08-26T14:18:38-07:00 (3 years, 1 month ago)
    
    OS-5634 system-log SMF method has existential confusion
    Reviewed by: Alex Wilson <alex.wilson@joyent.com>
    Reviewed by: Josh Wilsdon <jwilsdon@joyent.com>

diff --git a/overlay/generic/lib/svc/method/system-log b/overlay/generic/lib/svc/method/system-log
old mode 100644
new mode 100755
index f8a60a36..25f67919
--- a/overlay/generic/lib/svc/method/system-log
+++ b/overlay/generic/lib/svc/method/system-log
@@ -103,20 +103,21 @@ fi
 
 if [ -f /etc/rsyslog.conf ]; then
 	if [ ! -x /usr/sbin/rsyslogd ]; then
-		echo "/etc/rsyslog.conf is missing. Exiting."
+		echo "Executable /usr/sbin/rsyslogd not found or not executable. Exiting."
 		exit $SMF_EXIT_ERR_CONFIG
 	fi
 else
 	#
 	# Fall back to old syslogd
 	#
+	echo "/etc/rsyslog.conf is missing. Using syslogd instead."
 	if [ ! -f /etc/syslog.conf ]; then
 		echo "/etc/syslog.conf is missing. Exiting."
 		exit $SMF_EXIT_ERR_CONFIG
 	fi
 
 	if [ ! -x /usr/sbin/syslogd ]; then
-		echo "Executable /usr/sbin/syslogd not found. Exiting"
+		echo "Executable /usr/sbin/syslogd not found or not executable. Exiting."
 		exit $SMF_EXIT_ERR_CONFIG
 	fi
 fi
